cfme/middleware/__init__.py

```diff
@@ -27,6 +31,17 @@ def _on_detail_page(self):
         return sel.is_displayed('//h1[contains(., "{} (Summary)")]'.format(self.name))
 
 
+import_form = Form(
+    fields=[
+        ("file_select", "#upload_file"),
```

Comment:
Please use the FileInput, it is designed to control one of the "nastier" selenium background magic features ...